pkg/graph/schema.graphql

```diff
@@ -1015,8 +1015,27 @@ type SystemIntake {
   cedarSystemId: String
   documents: [SystemIntakeDocument!]!
   hasUiChanges: Boolean
+  itGovTaskStatuses:  ITGovTaskStatuses!
 }
+type ITGovTaskStatuses {
+  requesterStatuses: ITGovTaskStatusesRequester!
+  adminStatuses: ITGovTaskStatusesAdmin!
 
+}
+type ITGovTaskStatusesRequester {
+  intakeFormStatus: ITGovIntakeStatusReq!
+  feedbackFromInitialReviewStatus: ITGovFeedbackStatusReq!
+  bizCaseDraftStatus: ITGovDraftBuisnessCaseStatusReq!
+  grtMeetingStatus: ITGovGRTStatusReq!
+  bizCaseFinalStatus: ITGovFinalBuisnessCaseStatusReq!
+  grbMeetingStatus: ITGovGRBStatusReq!
+  decisionAndNextStepsStatus: ITGovDecisionStatusReq!
+}
+
+# TODO: to be implemented in a future ticket
```

Comment:
Do we have the ticket number for this?

Reply:
Thanks Dylan! Yes, we have quite a few tickets for these (one for each status) it starts with [EASI-2957](https://jiraent.cms.gov/browse/EASI-2957) and there are various other tickets up to EASI-2962

Reply:
I see, looks like each ticket covers both the requester view and admin view, if they're applicable. Probably don't need to specify all the ticket numbers here, then.